programs/virtual-curve/src/instructions/migration/meteora_damm/meteora_damm_lock_lp_token.rs

```diff
@@ -104,6 +110,13 @@ impl<'info> MigrateMeteoraDammLockLpTokenCtx<'info> {
 pub fn handle_migrate_meteora_damm_lock_lp_token_for_partner<'info>(
     ctx: Context<'_, '_, '_, 'info, MigrateMeteoraDammLockLpTokenCtx<'info>>,
 ) -> Result<()> {
+    let virtual_pool = ctx.accounts.virtual_pool.load()?;
+
+    require!(
```

Comment:
^Ditto for possible backward compatibility

Reply:
Ditto^